connexion/exceptions.py

```diff
@@ -23,11 +21,6 @@ def __init__(self, status=400, title=None, detail=None, type=None,
         self.headers = headers
         self.ext = ext
 
-    def to_problem(self):
-        return problem(status=self.status, title=self.title, detail=self.detail,
-                       type=self.type, instance=self.instance, headers=self.headers,
-                       ext=self.ext)
-
```

Comment:
I think we should leave this here with a deprecation warning. Removing this here is not backwards compatible.

@hjacobs What is the policy on connexion deprecations for connexion? ProblemException.to_problem was one of the primary interfaces for creating problems. It is no longer necessary with this change, but I would hate to break any code that depends on it without either a deprecation or a major version bump. Thoughts?

Reply:
Yes, let's leave it here with deprecation note (and create a follow-up issue to clean up in a future release)

Reply:
Added it back with a deprecated warning. Let me know if the message should be changed.